pkg/http/api.go

```diff
@@ -1050,6 +1050,90 @@ func UpdateConfig(host string, verifyTLS bool, apiKey string, project string, co
 	return info, Error{}
 }
 
+func UpdateConfigInheritable(host string, verifyTLS bool, apiKey string, project string, config string, inheritable bool) (models.ConfigInfo, Error) {
+	postBody := map[string]interface{}{"inheritable": inheritable}
+	body, err := json.Marshal(postBody)
+	if err != nil {
+		return models.ConfigInfo{}, Error{Err: err, Message: "Invalid config info"}
+	}
+
+	var params []queryParam
+	params = append(params, queryParam{Key: "project", Value: project})
+	params = append(params, queryParam{Key: "config", Value: config})
+
+	url, err := generateURL(host, "/v3/configs/config/inheritable", params)
+	if err != nil {
+		return models.ConfigInfo{}, Error{Err: err, Message: "Unable to generate url"}
+	}
+
+	statusCode, _, response, err := PostRequest(url, verifyTLS, apiKeyHeader(apiKey), body)
+	if err != nil {
+		return models.ConfigInfo{}, Error{Err: err, Message: "Unable to update config", Code: statusCode}
+	}
+
+	var result map[string]interface{}
+	err = json.Unmarshal(response, &result)
+	if err != nil {
+		return models.ConfigInfo{}, Error{Err: err, Message: "Unable to parse API response", Code: statusCode}
+	}
+
+	configInfo, ok := result["config"].(map[string]interface{})
+	if !ok {
+		return models.ConfigInfo{}, Error{Err: fmt.Errorf("Unexpected type parsing config info, expected map[string]interface{}, got %T", result["config"]), Message: "Unable to parse API response", Code: statusCode}
+	}
+	info := models.ParseConfigInfo(configInfo)
+	return info, Error{}
+}
+
+func UpdateConfigInherits(host string, verifyTLS bool, apiKey string, project string, config string, inherits string) (models.ConfigInfo, Error) {
+	inheritsObj := []models.ConfigDescriptor{}
+
+	if len(inherits) > 0 {
+		configDescriptors := strings.Split(inherits, ",")
+		for _, cd := range configDescriptors {
+			parts := strings.SplitN(cd, ".", 2)
```

Comment:
minor: If you remove the split limiter, we show the config descriptor error for more malformed strings. e.g. `database.dev.dev`

```diff
diff --git a/pkg/http/api.go b/pkg/http/api.go
index 92f7dd9..0414a58 100644
--- a/pkg/http/api.go
+++ b/pkg/http/api.go
@@ -1091,7 +1091,7 @@ func UpdateConfigInherits(host string, verifyTLS bool, apiKey string, project st
 	if len(inherits) > 0 {
 		configDescriptors := strings.Split(inherits, ",")
 		for _, cd := range configDescriptors {
-			parts := strings.SplitN(cd, ".", 2)
+			parts := strings.Split(cd, ".")
 			if len(parts) != 2 {
 				return models.ConfigInfo{}, Error{Message: "Config descriptors must match the format \"projectSlug.configName\""}
 			}


```